### DISCUSSION TITLE
Used array parameter in middleware

### DISCUSSION THREAD

**Discussion by youra-h** [nutgram/nutgram#384] [General]
I tried to add a processed function to the middleware using an array, but an error was thrown. As a result, after looking at the code, I came to the conclusion that the correct way to add an array looks like this:
wrong way
```php
$bot->middleware([$this, 'test']);
```
correctly
```php
$bot->middleware([[$this, 'test']]);
```
Is it right to do so?

**Comment by Lukasss93** [comment]
This [PR](https://github.com/nutgram/nutgram/pull/353/files) has broken this way of declaring a middleware.
How will we proceed? @SergiX44
